### ISSUE TITLE
migrate travis-ci to github workflow

### ISSUE THREAD

**Issue by zhfeng** [apache/servicecomb-pack#693]
It looks like travis-ci is suck and can we use github workflow ?

**Comment by WillemJiang** [comment]
Yeah, we need to move to github action.

**Comment by coolbeevip** [comment]
Open source accounts, as always, will be completely free under travis-ci.com, Maybe we can gradually move to travis-ci.com

https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/

**Comment by chanjarster** [comment]
> Open source accounts, as always, will be completely free under travis-ci.com, Maybe we can gradually move to travis-ci.com
> 
> https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/

No, it's not completely free, travis-ci.com consume 'credits' for each build. But you can request monthly OSS credits for OSS project.

https://docs.travis-ci.com/user/billing-faq/#what-if-i-am-building-open-source

**Comment by coolbeevip** [comment]
I created a JIRA https://issues.apache.org/jira/browse/SCB-2199 for that

**Comment by WillemJiang** [comment]
Now we just switch to GithubAction. It's time to consider to remove the travis setting now.

**Comment by coolbeevip** [comment]
Migrate Travis-ci to GitHub workflow completed.

* Master pr passed
* Master push passed
* Deploy snapshot passed (Run at 2 during night UTC)

Related #698 #700 #702 #703 #707 #708 #709
